digiroad2-oracle/src/main/scala/fi/liikennevirasto/digiroad2/dao/VerificationDao.scala

```diff
@@ -115,6 +115,7 @@ class VerificationDao {
         join property p on p.asset_type_id = a.asset_type_id and p.property_type = 'checkbox' and p.public_id = 'suggest_box'
         join multiple_choice_value mcv on mcv.asset_id = a.id and mcv.property_id = p.id
         join enumerated_value ev on ev.property_id = p.id and ev.value = 1 and ev.id = mcv.enumerated_value_id and  lrm.link_id in (select id from #$idTableName)
```

Comment:
This part " and  lrm.link_id in (select id from #$idTableName)"
Shouldn't be at WHERE clause ?

Reply:
I left it as it was. No problem when executing the query.

Reply:
Why not at line 114?
`scala

join lrm_position lrm on lrm.id = al.position_id and lrm.link_id in (select id from #$idTableName)
`

Reply:
Let's keep it like it is, it's working and already has reverted to different ways two times and the final result was always this.